apps/client/src/app/services/import-activities.service.ts

```diff
@@ -15,6 +15,7 @@ import { catchError } from 'rxjs/operators';
 })
 export class ImportActivitiesService {
   private static ACCOUNT_KEYS = ['account', 'accountid'];
+  private static COMMENT_KEYS = ['comment', 'note']
```

Comment:
Looks like a semicolon is missing here... 🙂

Reply:
ok, This time I have used prettier to fix all the linting issue related to my code. I hope things will be fine now... 🤞